labels/labels.json

```diff
@@ -91,10 +95,6 @@
     "name": "prio: P4 nice-to-have",
     "color": "#fde2b4"
   },
-  {
-    "name": "prio: +urgent",
-    "color": "#bf4639"
-  },
```

Comment:
This was added intentionally with the purpose of being used as an additional urgency-indicating label (as being marked by the "+" sign). The other labels are capturing the importance of a task. I didn't want to introduce a complete additional category with leveled urgencies, so this was kind of a practical compromise.

Would close this, feel free to re-take on the discussion if something is unclear.